fireant/tests/mock_dataframes.py

```diff
@@ -31,7 +33,7 @@ def rollup(dataframe, levels):
     'd': 'D',
     'y': 'Y',
     'z': 'Z',
-    np.nan: 'Total',
+    '_total': 'Total',
```

Comment:
could you please use the constant for this here and everywhere else in the tests?